installscripts/dockerfiles/jenkins-ce/Dockerfile

```diff
@@ -5,20 +5,20 @@ MAINTAINER JazzOSS Team
 ARG goVersion=1.10.3
 # Switching to root to configure the image with system packages
 USER root
-RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
-python3.6 python3-pip build-essential libssl-dev libffi-dev python3-dev && curl https://bootstrap.pypa.io/get-pip.py | python && curl https://bootstrap.pypa.io/get-pip.py | python3 && pip install --upgrade awscli && \
+RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less \
+python3.5 python3-dev python3-pip build-essential libssl-dev libffi-dev && curl https://bootstrap.pypa.io/get-pip.py | python && curl https://bootstrap.pypa.io/get-pip.py | python3 && pip install --upgrade awscli && \
```

Comment:
@rajeevr2715 So our new release will work with python3.5 and not python3.6?

Reply:
This has to work with Python3.6. Please test and confirm. If not then this needs to be fixed

Reply:
@suryajak By default it will have python 3.5, but I have updated the source package repository to fetch python 3.6 in docker container..
@svidhani  Validated and its working..